bundles/pubsub/pubsub_admin_tcp/src/pubsub_tcp_handler.c

```diff
@@ -1187,6 +1245,32 @@ char *pubsub_tcpHandler_get_interface_url(pubsub_tcpHandler_t *handle) {
     }
     return url;
 }
+//
+// get interface URL
+//
+char *pubsub_tcpHandler_get_connection_url(pubsub_tcpHandler_t *handle) {
+    hash_map_iterator_t iter =
```

Comment:
Missing `celixThreadRwlock_writeLock(&handle->dbLock);` in this function or in the caller function `pubsub_tcpTopicSender_url`

Reply:
Done